qmxgraph/widget.py

```diff
@@ -396,10 +396,15 @@ def _on_drop(self, event):
 
             # Refer to `mime.py` for docs about format
             version = parsed['version']
-            if version == 1:
-                x = event.pos().x()
-                y = event.pos().y()
+            if version not in (1, 2):
```

Comment:
Ok.
Let me know if there are any other problem. I have tackled the other issues.